### DISCUSSION TITLE
Portfolio link to add your climate coding challenge to your profile

### DISCUSSION THREAD

**Discussion by VictoriAragon** [earthlab-education/Earth-Analytics-AY24#451] [Q&A]
Does anyone have the format to use to put your portfolio climate challenge results on your profile page?

**Comment by eculler** [comment]
Hi @VictoriAragon, check out the [video on the textbook](https://cu-esiil-edu.github.io/esiil-learning-portal/foundations/notebooks/01-climate/climate-31-overview.html) for a walk-through of how to do this. I suggest exporting your notebook from Codespaces as HTML, and then uploading that to your site. You can then add that to your webpage as a link if you like.

**Reply by VictoriAragon** [reply]
Thank you Elsa, this is helpful
